pages/common/ruby.md

```diff
@@ -0,0 +1,23 @@
+# ruby
+
+> Ruby programming language interpreter.
+
+- Open an Interactive Ruby Shell (REPL):
+
+`irb`
+
+- Execute a Ruby script:
+
+`ruby {{script.rb}}`
+
+- Execute a single command in the command line:
+
+`ruby -e {{command}}`
+
+- Check if the given Ruby script is written correctly (Syntax check):
```

Comment:
Suggest changing it to - `Perform a syntax of the given Ruby script`

Reply:
Do you mean `Perform a syntax check of the given Ruby script`? Or do you want to just say syntax? How about `Check syntax errors on a Ruby script` because then it would follow the same pattern as the tldr for "perl"

Reply:
Oops yea I meant syntax check. Sure your version sounds good too. Please make it `Check for syntax errors on a given Ruby script`